go.mod

```diff
@@ -117,4 +117,4 @@ require (
 )
 
 //replace github.com/ethereum/go-ethereum => github.com/certifaction/go-ethereum v1.10.3-wasm
-//replace github.com/herumi/bls-go-binary => github.com/0chain/bls-go-library-zboxmobile v1.26.1
+replace github.com/herumi/bls-go-binary => github.com/cnlangzi/bls-go-binary v1.26.1
```

Comment:
Any changes in `github.com/cnlangzi/bls-go-binary'? if so, let's fork the bls-go-binary to 0chain org, and commit the changes.

Reply:
- fixed simulator issue. see detail on https://github.com/herumi/bls-go-binary/pull/9
- uploaded pre-complied binaries for iOS and iOS simulator on `binary` branch, and create a tag for it.

Reply:
fixed